docs/tempo/website/setup/set-up-test-app.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Set up a test app for a Tempo cluster
+menuTitle: Set up a test application for a Tempo cluster
+description: Learn how to set up a test app for your Tempo cluster and visualize data.
+weight: 400
+---
+
+# Set up a test application for a Tempo cluster
+
+Once you've set up a Grafana Tempo cluster, you need to write some traces to it and then query the traces from within Grafana.
+
+## Before you begin
+
+You'll need:
+
+* Grafana 9.0.0 or higher
+* Microservice deployments require the Tempo querier URL, for example: `http://query-frontend.tempo.svc.cluster.local:3200`
+
+Refer to [Deploy Grafana on Kubernetes](https://grafana.com/docs/grafana/latest/setup-grafana/installation/kubernetes/#deploy-grafana-on-kubernetes) if you are using Kubernetes.
+Otherwise, refer to [Install Grafana](https://grafana.com/docs/grafana/latest/installation/) for more information.
+
+## Set up `remote_write` to your Tempo cluster
+
+To enable writes to your cluster:
+
+1. Add a `remote_write` configuration snippet to the configuration file of an existing Grafana Agent.
+
+   If you do not have an existing traces collector, refer to [Set up with Grafana Agent](https://grafana.com/docs/agent/latest/set-up/).
+   For Kubernetes, refer to the [Grafana Agent Traces Kubernetes quick start guide](https://grafana.com/docs/grafana-cloud/kubernetes-monitoring/agent-k8s/k8s_agent_traces/).
+
+   The example agent Kubernetes ConfigMap configuration below opens many trace receivers (note that the remote write is onto the Tempo cluster using OTLP gRPC):
+
+    ```yaml
+    kind: ConfigMap
+    metadata:
+      name: grafana-agent-traces
+    apiVersion: v1
+    data:
+      agent.yaml: |
+        traces:
+            configs:
+              - batch:
+                    send_batch_size: 1000
+                    timeout: 5s
+                name: default
+                receivers:
+                    jaeger:
+                        protocols:
+                            grpc: null
+                            thrift_binary: null
+                            thrift_compact: null
+                            thrift_http: null
+                    opencensus: null
+                    otlp:
+                        protocols:
+                            grpc: null
+                            http: null
+                    zipkin: null
+                remote_write:
+                  - endpoint: <tempoDistributorServiceEndpoint>
+                    insecure: true  # only add this if TLS is not required
```

Comment:
"...TLS is not used" perhaps.  Folks who need/want TLS know they need/want TLS I suppose.

Reply:
I've updated the comment to say "TLS is not used"